src/containers/Member.js

```diff
@@ -102,8 +106,42 @@ const MemberPage = ({
     return null
   }
 
+  const getSiteTitle = pathname => {
+    switch (pathname) {
```

Comment:
Could we match the pathname with [matchPath](https://v5.reactrouter.com/web/api/matchPath) function provided by react-router-dom

Reply:
fixed